process/block/metablock.go

```diff
@@ -1740,7 +1772,8 @@ func (mp *metaProcessor) getLastFinalizedMetaHashForShard(shardHdr *block.Header
 		currentHdr = prevShardHdr
 	}
 
-	return nil, nil, process.ErrLastFinalizedMetaHashForShardNotFound
+	//TODO: get header hash from last epoch start metablock
+	return nil, nil, nil
```

Comment:
Is it ok to return all nils? Will the method that uses these results not set some values that should not be nil to nil?

Reply:
will be done when the start epoch is implemented